docs/user-manual.md

```diff
@@ -102,6 +64,64 @@ $ kustomize build github.com/cybozu-go/meows/config/controller?ref=${MEOWS_VERSI
 
 After deploying the controller, let's create `RunnerPool` resources to register self-hosted runners.
 
+In meows, you can change the GitHub Credential for each RunnerPool.
+In other words, you need to create a Secret that records the credential in the RunnerPool's namespace before.
+
+### Creating RunnerPool's namespace
+
+If you have not created a RunnerPool's namespace yet, please create as follows.
```

Comment:
I don't think this is grammatically correct.
How about `please create follows` or `please create it as follows`.